templates/microservice/serverless.yml

```diff
@@ -1,53 +1,54 @@
-service: general
```

Comment:
There were some things going on in here that I didn't fully understand. Previously, there was a requirement in the generated project requiring a leo_cli_config.js and running some functions that seem to do what serverless-convention does. I've since removed these refs from this template, but I've left the original code in serverless-leo.

Reply:
This is a good call to remove the default `general` service.